Library/Formula/postgresql.rb

```diff
@@ -82,6 +90,15 @@ def install
 
     system "./configure", *args
     system "make", "install-world"
+
+    # Now build and install pgroonga as optional resource
+    if build.with? "pgroonga"
+      resource("pgroonga").stage do
+        ENV.append_path "PATH", "#{Formula["postgresql"].bin}"
```

Comment:
Doesn't need the surrounding `""` here. Should work as `ENV.append_path "PATH", Formula["postgresql"].bin`